src/text2.c

```diff
@@ -3467,15 +3467,15 @@ void xxx_draw_string(UnkTextStruct1 *strArr, s32 x, s32 y, const u8 *str, u32 wi
         else if (sp.unk34 == 0x1B) {
             break;
         }
-        else if (sp.unk34 == 0xD || sp.unk34 == 0xA) {
+        else if (sp.unk34 == 0xD || sp.unk34 == '\n') {
```

Comment:
0xD = `'\r'`

Reply:
0x1B is `'\e'` (Escape character)
0x1D has no short escape in C, but could be `'\x1D'`. It's the ASCII group separator.